plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
+ZONE_RETRY = 60 * 60
```

Comment:
I might be way off base or misunderstanding the values, but it seems odd to have the retry interval set to an hour when the refresh interval is set to 5 minutes.  I would normally expect the refresh number to be some multiple of the retry.